M2/Macaulay2/packages/Macaulay2Doc/doc_tables.m2

```diff
@@ -233,15 +233,16 @@ doc ///
   (pairs, HashTable)
   (pairs, Dictionary)
   (pairs, BasicList)
+  (pairs, Thing)
  Headline
   list the pairs in a hash table, dictionary, or basic list
  Usage
   pairs x
  Inputs
-  x:HashTable
-   or @TO Dictionary@ or @TO BasicList@
+  x:{HashTable, Dictionary, BasicList}
+   or any instance of a class with an @TO iterator@ method installed
  Outputs
-  L:List
+  L:{List, Iterator}
```

Comment:
I think it might be better to separate the documentation of the iterator form, ideally using `Synopsis`.

Reply:
Cool -- I didn't know about `Synopsis`!  Definitely useful for stuff like this.